cmd/autoscaler/main.go

```diff
@@ -218,7 +219,10 @@ func uniScalerFactoryFunc(endpointsInformer corev1informers.EndpointsInformer,
 			return nil, err
 		}
 
-		return scaling.New(decider.Namespace, decider.Name, metricClient, endpointsInformer.Lister(), &decider.Spec, ctx)
+		pc := resources.NewScopedEndpointsCounter(endpointsInformer.Lister(),
+			fake.TestNamespace, fake.TestService)
```

Comment:
Why are we not using the actual service/ns names here?

Reply:
ugh. I don't know how this made it here.